cmd/hamctl/command/root.go

```diff
@@ -72,18 +72,17 @@ func NewRoot(version *string) (*cobra.Command, error) {
 	command.AddCommand(
 		NewCompletion(command),
 		NewDescribe(&client, &service),
-		NewPolicy(&client, &service, gitConfigAPI),
+		NewPolicy(&client, &service),
 		NewPromote(&client, &service, releaseClient),
 		NewRelease(&client, &service, loggerFunc, releaseClient),
 		NewRollback(&client, &service, loggerFunc, SelectRollbackReleaseFunc, releaseClient),
 		NewStatus(&client, &service),
 		NewVersion(*version),
+		Login(authenticator),
 	)
 	command.PersistentFlags().DurationVar(&client.Timeout, "http-timeout", 120*time.Second, "HTTP request timeout")
 	command.PersistentFlags().StringVar(&client.BaseURL, "http-base-url", "", "address of the http release manager server")
-	command.PersistentFlags().StringVar(&client.Metadata.AuthToken, "http-auth-token", "", "auth token for the http service")
```

Comment:
To make a gradual roll out we might consider only deprecating the flags and remove them in a later release? That way we can update any use of them without everything breaking.

Just a thought 👌

Reply:
I just checked and this specific flag isn't used. We should be all good. The server middleware supports both authentication mechanisms, so users with an older hamctl will still have access during the rollout.